JamGame/Assets/Scripts/Employee/ExtendedInfo/View.cs

```diff
@@ -0,0 +1,169 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Linq;
+using Common;
+using Employee.Personality;
+using Sirenix.OdinInspector;
+using TMPro;
+using UnityEngine;
+using UnityEngine.AddressableAssets;
+using UnityEngine.ResourceManagement.ResourceLocations;
+
+namespace Employee.ExtendedInfo
+{
+    [AddComponentMenu("Scripts/Employee/ExtendedInfo/Employee.ExtendedInfo.View")]
+    public class View : MonoBehaviour
+    {
+        private Camera cam;
+
+        private PersonalityImpl personality;
+        private EmployeeImpl employee;
+
+        [Required]
+        [SerializeField]
+        [ChildGameObjectsOnly]
+        private TMP_Text name_text;
+
+        [SerializeField]
+        private AssetLabelReference quirkViewsLabel;
+
+        [SerializeField]
+        private AssetLabelReference buffViewsLabel;
+
+        private static Dictionary<string, IResourceLocation> quirkModelViewMap = new();
+        private static Dictionary<string, IResourceLocation> buffModelViewMap = new();
+
+        [SerializeField]
+        [Required]
+        private Transform quirksContainer;
+
+        [SerializeField]
+        [Required]
+        private Transform buffsContainer;
+
+        private List<BuffView> instantiatedBuffViews = new();
+
+        private void OnEnable()
+        {
+            employee.AppliedBuffsChanged += OnBuffsChanged;
+        }
+
+        private void OnDisable()
+        {
+            employee.AppliedBuffsChanged -= OnBuffsChanged;
+        }
+
+        private void Awake()
+        {
+            cam = Camera.main;
+
+            personality = GetComponentInParent<PersonalityImpl>();
+            employee = GetComponentInParent<EmployeeImpl>();
```

Comment:
We can move `ExtendedInfo` into Employee and make this dependencies `[Required]` and just turn on/off `ExtendedInfo`